src/components/LocationAutocompleteInput/LocationAutocompleteInputImpl.js

```diff
@@ -526,6 +527,8 @@ class LocationAutocompleteInputImplementation extends Component {
         <div className={iconClass}>
           {this.state.fetchingPlaceDetails ? (
             <IconSpinner className={css.iconSpinner} />
+          ) : useAlternateIcon ? (
+            <IconLocation />
```

Comment:
Make this more customizable. Don't pass in flag `useAlternateIcon`, but `customIcon`.

In [LocationAutocompleteInput](https://github.com/sharetribe/web-template/blob/main/src/components/LocationAutocompleteInput/LocationAutocompleteInput.js#L15): Add JSDoc comment:
`* @param {ReactNode} props.customIcon`

When spreading props on this file:
`  customIcon: CustomIcon,`
I.e. map the property to be a proper React component (which start with a capital letter).

Here
```suggestion
          ) : CustomIcon ? (
            <CustomIcon />
```

The end result is that IconLocation should be passed to this component on **_SearchCTA/FilterLocation.js_**